src/components/Context/index.js

```diff
@@ -17,7 +17,9 @@ import injectPlaceContext from "./injectPlaceContext";
 import injectTimestamp from "./injectTimestamp";
 import implementationDetails from "./implementationDetails";
 import createComponent from "./createComponent";
+import window from "../../utils/window";
 import { arrayOf, string } from "../../utils/validation";
+import ObjectKeys from "../../utils/Object.keys";
```

Comment:
I'd prefer to use camel case here "objectKeys" for the variable name, and objectKeys.js for the file name.